lib/constructs/cost-api-implementation.ts

```diff
@@ -79,9 +79,7 @@ export class CostApiImplementation extends Construct implements ICostApiImplemen
 
     // secrets permissions
     cspConfig.grantRead(this.costRequestFn);
-    props.idp?.addClient(new idp.IdentityProviderLambdaClient("CspWriteCost", this.costRequestFn), [
-      "atat/write-portfolio",
-    ]);
+    props.idp?.addClient(new idp.IdentityProviderLambdaClient("CspWriteCost", this.costRequestFn), ["atat/read-cost"]);
```

Comment:
Should `CspWriteCost` also be changed?
```suggestion
    props.idp?.addClient(new idp.IdentityProviderLambdaClient("CspReadCost", this.costRequestFn), ["atat/read-cost"]);
```

Reply:
Hmm curious to see what this does in CloudFormation -- let's roll with it!